gamehub-backend/src/main/java/com/example/gamehubbackend/controllers/UserController.java

```diff
@@ -21,18 +22,23 @@ public List<User> getAllUsers() {
         return userService.getAllUser();
     }
 
+    @GetMapping("/me")
+    public UserResponse getLoggedInUser() {
+        return userService.getLoggedInUser();
+    }
+
     @GetMapping("/{id}")
     public User getUserById(@PathVariable String id) {
```

Comment:
should this and getAllUsers not also return only UserResponse? not sure

Reply:
In theory i think yes, but this endpoint is at this point not used.